src/package.json

```diff
@@ -14,8 +14,7 @@
     "react-ga": "~2.7.0",
     "react-phone-input-2": "^2.13.5",
     "react-router-dom": "^5.0.0",
-    "reconnecting-websocket": "^4.4.0",
-    "rxjs": "^6.5.4"
+    "reconnecting-websocket": "^4.4.0"
```

Comment:
I think the removal of rxjs will generate some lockfile changes when you run `npm i`, which should be committed.

Reply:
I tried that. It actually seemed to add content, not remove it (maybe those are from another change?). I think `rxjs` is a sub-dependency of a couple other libraries.